cilium/load_balancer/docs/CiliumL4LBSetup.md

```diff
@@ -72,24 +72,24 @@ The demo will use ```40.1.1.1``` as the LB VIP.
 
 The following steps describe how to setup the LB node VM:
-1. Install [Clang 10.0](https://github.com/llvm/llvm-project/releases/tag/llvmorg-10.0.0). Download ``LLVM-10.0.0-win64.exe``
-2. Ensure Clang is added to system PATH.
+1. Install [Clang 11.0.1](https://github.com/llvm/llvm-project/releases/tag/llvmorg-11.0.1). Download ``LLVM-11.0.1-win64.exe``.
+2. Ensure Clang is added to system PATH. Check ``"clang --version"``.
 3. Enable test signing by running command ``"bcdedit -set testsigning on"``. This command needs to be run as Administrator.
 4. Create a folder ``C:\ebpf`` on the LB node.
-5.  Copy all the files from the 2 build directories (for `ebpf-for-windows` and `ebpf-for-windows-demo`) mentioned in the previous section to ``C:\ebpf``
-5. Go to ``C:\ebpf`` and execute ``"install_ebpf.bat"``. This will install the ebpf-for-Windows framework on the LB node.
+5.  Copy all the files from the build directory (`ebpf-for-windows-demo`) mentioned in the previous section to ``C:\ebpf``
+6. Install ``ebpf-for-windows.<version>`` on the LB node using Windows Installer Package from [here](https://github.com/microsoft/ebpf-for-windows/releases). Download ``ebpf-for-windows.<version>.msi``. Also download and install ``Microsoft Visual C++ Redistributable (x64)`` exe. ebpf-for-windows will be installed in ``C:\Program Files\ebpf-for-windows``
 
 #### Setup Windows Backend Node(s)
 
-This section describes how to configure the Windows backend nodes. For DSR mode, backend nodes will receive an IP-in-IP packet, and we need some eBPF program to decap those packets. On Windows backend nodes, the demo will use an XDP sample program (ebpf-for-windows/decap_permit_packet.c at master · microsoft/ebpf-for-windows (github.com) to decap IP-in-IP packets. The sample is present in eBPF-for-Windows project. Following are the steps to setup the Windows backend nodes:
+This section describes how to configure the Windows backend nodes. For DSR mode, backend nodes will receive an IP-in-IP packet, and we need some eBPF program to decap those packets. On Windows backend nodes, the demo will use an XDP sample program ebpf-for-windows/decap_permit_packet.c at master · microsoft/ebpf-for-windows (github.com) to decap IP-in-IP packets. The sample is present in eBPF-for-Windows project. Following are the steps to setup the Windows backend nodes:
 
 1. Enable test signing on the node by running command ``"bcdedit -set testsigning on"``. This command needs to be run as Administrator.
-2. Create a folder ``C:\ebpf`` and copy the all the files from the `ebpf-for-windows` build directory mentioned above to ``C:\ebpf``
+2. Create a folder ``C:\ebpf`` and copy the all the files from the `ebpf-for-windows` build directory in dev machine to ``C:\ebpf``

```

Comment:
This line should be removed, as we need to use the MSI to install ebpf.

Reply:
but the decap_permit_packet.o is only present from the ebpf-for-windows build, not in the .msi installation.

Reply:
Please file a bug in the ebpf-for-windows repo then to get it added to the Testing component then.

Reply:
Revised the doc with the current behavior (ie, to build ebpf-for-windows and bring in only one .o file).

Created an issue:
  Add decap_permit_packet.o in ebpf-for-windows.<version>.msi #2642